### ISSUE TITLE
peachpie-wordpress websocket plugin?

### ISSUE THREAD

**Issue by mkmydrive** [iolevel/wpdotnet-sdk#120]
I want to add a websocket chatroom for peachpie-wordpress,is there a plugin?

**Comment by jakubmisek** [comment]
Only if there is any on https://wordpress.org/plugins/.

**Comment by mkmydrive** [comment]
Yes,the most welcome plugin is  here. https://wordpress.org/plugins/bp-better-messages/
